### PR TITLE
Enable EnemyCD color class border

### PR DESCRIPTION
**Pull request by AlbertDuval**

### CODE REVIEW

File: ShestakUI/Modules/Cooldowns/EnemyCD.lua [hunk 1]
```diff
@@ -102,13 +102,17 @@ local CreateIcon = function()
...
-local StartTimer = function(name, sID)
+local StartTimer = function(sGUID, sID)
 	local _, _, texture = GetSpellInfo(sID)
 	local icon = CreateIcon()
 	icon.Texture:SetTexture(texture)
 	icon.Texture:SetTexCoord(0.1, 0.9, 0.1, 0.9)
 	icon.endTime = GetTime() + T.EnemySpells[sID]
-	local color = (CUSTOM_CLASS_COLORS or RAID_CLASS_COLORS)[select(2, UnitClass(name))]
+	local locClass, _, _, _, _, name, _ = GetPlayerInfoByGUID(sGUID)
+	local color = NULL
```
Comment (Wetxius) [inline ShestakUI/Modules/Cooldowns/EnemyCD.lua:112]: NULL leak global?

File: ShestakUI/Modules/Cooldowns/EnemyCD.lua [hunk 2]
```diff
@@ -102,13 +102,17 @@ local CreateIcon = function()
...
 
-local StartTimer = function(name, sID)
+local StartTimer = function(sGUID, sID)
 	local _, _, texture = GetSpellInfo(sID)
 	local icon = CreateIcon()
 	icon.Texture:SetTexture(texture)
 	icon.Texture:SetTexCoord(0.1, 0.9, 0.1, 0.9)
 	icon.endTime = GetTime() + T.EnemySpells[sID]
-	local color = (CUSTOM_CLASS_COLORS or RAID_CLASS_COLORS)[select(2, UnitClass(name))]
+	local locClass, _, _, _, _, name, _ = GetPlayerInfoByGUID(sGUID)
```
Comment (Wetxius) [inline ShestakUI/Modules/Cooldowns/EnemyCD.lua:111]: Should be without , _ in the end
local locClass, _, _, _, _, name = GetPlayerInfoByGUID(sGUID)

File: ShestakUI/Config/Settings.lua [hunk 3]
```diff
@@ -557,7 +557,7 @@ C["enemycooldown"] = {
 	["show_inpvp"] = false,						-- Show in bg zone
 	["show_inarena"] = true,					-- Show in arena zone
 	["show_inparty"] = false,					-- Show in party zone for allies
-	["class_color"] = false,					-- Enable classcolor border
+	["class_color"] = true,					-- Enable classcolor border
```
Comment (Wetxius) [inline ShestakUI/Config/Settings.lua:560]: It should be false anyway.

**Review by Wetxius — COMMENTED** [review]
It will show class color in bg and arena?

File: ShestakUI/Modules/Cooldowns/EnemyCD.lua (hunk 2 above)
Comment (AlbertDuval) [reply ShestakUI/Modules/Cooldowns/EnemyCD.lua:111]: The other informations gather by GetPlayerInfoByGUID are not interesting

File: ShestakUI/Config/Settings.lua (hunk 3 above)
Comment (AlbertDuval) [reply ShestakUI/Config/Settings.lua:560]: ok

**Pull request closed without merging**